src/components/Follow/FollowingList.tsx

```diff
@@ -24,10 +39,29 @@ const FollowingList = ({ src, size = '80px', fullName }: Props) => {
       }}>
       <ListItem>
         <ListItemAvatar>
-          <Avatar
-            src={src}
-            sx={{ width: size, height: size, marginRight: '0.5rem' }}
-          />
+          {isOnline ? (
+            <OnlineBadge
+              overlap='circular'
+              anchorOrigin={{ vertical: 'bottom', horizontal: 'right' }}
+              variant='dot'>
+              <Avatar
+                src={src}
+                sx={{ width: size, height: size }}
+                onClick={() => navigate(ROUTES.STORY_BOOK_BY_USER_ID(userId))}
+              />
+            </OnlineBadge>
+          ) : (
+            <OfflineBadge
+              overlap='circular'
+              anchorOrigin={{ vertical: 'bottom', horizontal: 'right' }}
+              variant='dot'>
+              <Avatar
+                src={src}
+                sx={{ width: size, height: size }}
+                onClick={() => navigate(ROUTES.STORY_BOOK_BY_USER_ID(userId))}
+              />
+            </OfflineBadge>
+          )}
```

Comment:
이거 줄일 수 있을 거 같은데 한번 같이 고민해 볼까요?

예를들어서 이렇게 할거같아요 저는

```suggestion
          <StateBadge
            isOnline={isOnline}>
            <Avatar
              src={src}
              sx={{ width: size, height: size }}
              onClick={() => navigate(ROUTES.STORY_BOOK_BY_USER_ID(userId))}
            />
          </StateBadge>
```

Reply:
수정 했습니다~